cuda_core/cuda/core/experimental/_module.py

```diff
@@ -278,6 +278,24 @@ def from_cubin(module: Union[bytes, str], *, symbol_mapping: Optional[dict] = No
         """
         return ObjectCode._init(module, "cubin", symbol_mapping=symbol_mapping)
 
+    @staticmethod
+    def from_ptx(module: Union[bytes, str], *, symbol_mapping: Optional[dict] = None) -> "ObjectCode":
+        """Create an :class:`ObjectCode` instance from an existing cubin.
+
+        Parameters
+        ----------
+        module : Union[bytes, str]
+            Either a bytes object containing the in-memory ptx code to load, or
+            a file path string pointing to the on-disk ptx file to load.
+        symbol_mapping : Optional[dict]
+            A dictionary specifying how the unmangled symbol names (as keys)
+            should be mapped to the mangled names before trying to retrieve
+            them (default to no mappings).
+        """
+        obj = ObjectCode._init(module, "cubin", symbol_mapping=symbol_mapping)
+        obj._code_type = "ptx"
```

Comment:
Untested but I think this should work
```suggestion
        obj = ObjectCode._init(module, "ptx", symbol_mapping=symbol_mapping)
```
Also, does taking `symbol_mapping` make sense? I have no idea how PTX handles C++ symbol mangling -- does it only use mangled symbols (I guess so)?